src/main/java/com/endava/soa_5ed/task1_basics/Ex1.java

```diff
@@ -0,0 +1,23 @@
+package com.endava.soa_5ed.task1_basics;
+
+import java.util.Scanner;
+
+public class Ex1 {
+    //    Write a method that converts a total number of seconds to hours, minutes, and seconds.
+    final static int toHours = 3600;
+    final static int toMinutes = 60;
+
+    public static void main(String[] args) {
+        Scanner inputSeconds = new Scanner(System.in);
+        System.out.print("Please enter desired number to convert: ");
+        int seconds = inputSeconds.nextInt();
+        System.out.println(convertTime(seconds));
+    }
+    static String convertTime(int seconds){
```

Comment:
Try to avoid declaring methods static, better to use objects.

Reply:
done, thanks